tests/test_convert.py

```diff
@@ -0,0 +1,23 @@
+import sys
+import unittest
+import os
+from tf2onnx import convert
```

Comment:
pep8: 1 line between sys imports and local imports

Reply:
added.